code/game/objects/items/stacks/medical.dm

```diff
@@ -298,13 +303,11 @@
 			to_chat(user, span_warning("You can't apply a splint there!"))
 			return
 
-		if(affecting.limb_status & LIMB_DESTROYED)
-			to_chat(user, span_warning("[user == M ? "You don't" : "[M] doesn't"] have \a [limb]!"))
-			return
-
-		if(affecting.limb_status & LIMB_SPLINTED)
-			to_chat(user, span_warning("[user == M ? "Your" : "[M]'s"] [limb] is already splinted!"))
-			return
```

Comment:
These checks aren't necessary because the limb itself handles them in its apply_splints

Reply:
does that mean you have the windup then the warning?

Reply:
You'll have the fumble delay first, then the cancel out if the limb is destroyed/splinted, and then the splinting do_after.

Reply:
so keeping the checks would be better no? why would you fumble to check if a splint is still here or destroy?

Reply:
Gross incompetence. You can also avoid this by looking at the person, which tells you if any limbs are splinted or missing.